### PR TITLE
Add initial Vagrant setup

### LINKED ISSUE #120: Create vagrant setup for hacking on datanommer
Create a Vagrant setup to help new contributors hack on datagrepper easier.

### PR DESCRIPTION
**Pull request by ryanlerch**
Add an initial vagrant setup for datanommer.

This installs the 3 packages into a virtualenv.

No configuration setup yet, as that is all still fedmsg based

Resolves: #120

### CODE REVIEW

File: devel/ansible/roles/datanommer/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    group: vagrant
+
+- name: Create the virtualenv and install poetry
+  pip:
+    name: poetry
+    virtualenv: /srv/venv
+  become: yes
+  become_user: vagrant
+
+- name: Tell poetry not to create a virtualenv
```
Comment (abompard) [inline devel/ansible/roles/datanommer/tasks/main.yml]: I'm not sure that's necessary, poetry should not create an additional virtualenv when it's running from a virtualenv already

Comment (abompard) [reply devel/ansible/roles/datanommer/tasks/main.yml]: Which means that we should source `/srv/venv/bin/activate` from the vagrant user's `.bashrc`.

**Review by abompard — CHANGES_REQUESTED** [review]

File: devel/ansible/roles/datanommer/tasks/main.yml (hunk 1 above)
Comment (ryanlerch) [reply devel/ansible/roles/datanommer/tasks/main.yml]: Fair enough -- was just looking at fasjson's vagrant setup, and in there we alias /srv/venv/bin/potery to poetry: https://github.com/fedora-infra/fasjson/blob/dev/ansible/roles/fasjson/files/.bashrc

does that achieve the same thing? or would  you rather activating the venv with source in the bashrc?

**Comment by ryanlerch** [comment]
Added two more commits to this one implemented the requested changes.

Will squash them and move out of WIP -- just keeping commits to make it easier to see what i have changed.

File: devel/ansible/roles/datanommer/tasks/main.yml (hunk 1 above)
Comment (abompard) [reply devel/ansible/roles/datanommer/tasks/main.yml]: I think it's pretty much the same. But it feels like enabling the virtualenv is closer to the "nominal path" than making an alias and setting the poetry configuration. Maybe it's just me :-) Both would probably work.

**Pull request merged**